app/javascript/src/utilities/toast.ts

```diff
@@ -0,0 +1,119 @@
+// import { Wrapper as Flash } from 'Common/components/Flash'
+
+const TIMEOUT = 5000
+
+export type TYPE = 'danger' | 'default' | 'info' | 'success' | 'warning'
+
+const TYPES = ['danger', 'default', 'info', 'success', 'warning']
```

Comment:
```suggestion
const TYPES = ['danger', 'default', 'info', 'success', 'warning']

export type TYPE =  typeof TYPES[number]
```

Not sure if it's exactly correct, but it would be nice to reduce duplication